app/views/user_mailer/suspended_create_account.html.erb

```diff
@@ -0,0 +1,13 @@
+<table class="spacer">
+  <tbody>
+    <tr>
+      <td class="s10" height="10px">
+        &nbsp;
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<p>
+<%= t('user_mailer.suspended_create_account.message', support_code: IdentityConfig.store.account_suspended_support_code, contact_number: IdentityConfig.store.idv_contact_phone_number) %>
```

Comment:
this is missing the `app_name` param:
```suggestion
<%= t('user_mailer.suspended_create_account.message', app_name: APP_NAME, support_code: IdentityConfig.store.account_suspended_support_code, contact_number: IdentityConfig.store.idv_contact_phone_number) %>
```

Reply:
Thanks!  I had to back into that because the linter caught the “Login.gov” in the translation file…  😐